tests/integration.rs

```diff
@@ -0,0 +1,495 @@
+use std::error::Error as StdError;
+use std::io::Read;
+use std::process::{Child, Command};
+use std::time::Duration;
+use std::{env, fs, io};
```

Comment:
Nit: move the `io` import to `std::io::{self, Read}`.

Reply:
Done